internal/conf/computed.go

```diff
@@ -136,7 +136,7 @@ func CanReadEmail() bool {
 // Deploy type constants. Any changes here should be reflected in the DeployType type declared in web/src/globals.d.ts:
 // https://sourcegraph.com/search?q=r:github.com/sourcegraph/sourcegraph%24+%22type+DeployType%22
 const (
-	DeployCluster       = "cluster"
+	DeployKubernetes    = "cluster"
```

Comment:
```suggestion
	DeployKubernetes    = "kubernetes"
```

I assume we'd want this otherwise these would show up in pings as "cluster" too?